legal-api/src/legal_api/services/business_details_version.py

```diff
@@ -500,7 +516,7 @@ def party_role_revision_json(filing, party_role_revision, is_ia_or_after) -> dic
         )
         if party_role_revision.related_colin_entity_id:
             party_revision = VersionedBusinessDetailsService.get_colin_entity_revision(
-                filing, party_role_revision.related_entity_id
+                filing, party_role_revision.related_colin_entity_id
```

Comment:
Think it's a bug. Fixed. Same as line 553. https://github.com/bcgov/lear/blob/414610846de8cdbad87e6cc682ab7db1f90a6c9e/legal-api/src/legal_api/services/business_details_version.py#L553